.circleci/config.yml

```diff
@@ -84,6 +84,7 @@ jobs:
             source venv/bin/activate
             pip install --upgrade pip wheel setuptools
             pip install -r requirements/default.txt -r requirements/test.txt
+            pip install pytest-mpl
```

Comment:
It might be worth adding a note here to remind others/our-future-selves why `pytest-mpl` is not yet in `requirements/test.txt`